README.md

```diff
@@ -51,37 +61,40 @@ try {
     wrapper.close();
 }
 ```
-Review the [example](https://github.com/signalfx/lambda-java/blob/master/src/java/com/signalfx/lambda/example/CustomHandler.java).
+2. For additional custom function signatures, review the [custom handler example](https://github.com/signalfx/lambda-java/blob/master/src/java/com/signalfx/lambda/example/CustomHandler.java).
```

Comment:
In case of the manual wrapper, it actually doesn't matter what the signature is. It is just a full working example (lines 53-63 are an extract). So maybe just: "See the complete example for manually wrapping a function" or sth like that?

Reply:
This has been updated.